plugins/modules/remote_management/ism/is_add_user.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+# -*- coding:utf-8 -*-
+
+# Copyright(C) 2020 Inspur Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: is_add_user
+version_added: "0.1.0"
+author:
+    - WangBaoshan (@ISIB-group)
+short_description: Create user
+description:
+   - Create user on Inspur server.
+options:
+    uname:
+        description:
+            - User name of new user.
+        type: str
+        required: true
+    upass:
+        description:
+            - User password of new user.
+        type: str
+        required: true
+    role_id:
+        description:
+            - user role id of new user.
+        default: NoAccess
+        choices: ['Administrator', 'Operator', 'Commonuser','OEM','NoAccess']
+        type: str
+    priv:
+        description:
+            - user access, select one or more from None/KVM/VMM/SOL.
+        type: str
+        required: true
+extends_documentation_fragment:
+- community.general.ism
+
+'''
+
+EXAMPLES = '''
+- name: add user test
+  hosts: ism
+  connection: local
+  gather_facts: no
+  vars:
+    ism:
+      host: "{{ ansible_ssh_host }}"
+      username: "{{ username }}"
+      password: "{{ password }}"
+
+  tasks:
+
+  - name: "add user"
+    is_add_user:
+      uname: "wbs"
+      upass: "admin"
+      role_id: "Administrator"
+      priv: "KVM,SOL"
+      provider: "{{ ism }}"
+'''
+
+RETURN = '''
+message:
+    description: messages returned after module execution
```

Comment:
```suggestion
    description: Messages returned after module execution.
```
There are more (probably also in other files).